scenarios/pravega-flink-connector-sql-samples/README.md

```diff
@@ -0,0 +1,46 @@
+# Flink Pravega Table API samples
+
+This module contains examples to demonstrate the use of FLink connector Table API and it uses [NY Taxi Records](http://www.nyc.gov/html/tlc/html/about/trip_record_data.shtml) to demonstrate the usage.  
```

Comment:
It is important that we properly reference the data we are copying. You may want to point specifically to the files that contain the data we are copying from this source.

Reply:
added reference to the file that we are using in the sample app